doc/tutorials/sagemaker/sme_init.ipynb

```diff
@@ -0,0 +1,300 @@
+{
```

Comment:
<p>We need to fix the naming of AWS_KEY and AWS_ACCESS_KEY, because the official names are for the first ACCES_KEY and for the second SECRET_KEY. Can also happen in another PR</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/data-science-sandbox/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I can just as well do this in this PR.</p>

Reply:
<p>From AWS glossary:</p><p> </p><p>access key</p><p> </p><p>The combination of an&nbsp;<a href="https://docs.aws.amazon.com/glossary/latest/reference/glos-chap.html#accesskeyID" target="_blank">access key ID</a>&nbsp;(for example,&nbsp;<code>AKIAIOSFODNN7EXAMPLE</code>) and a&nbsp;<a href="https://docs.aws.amazon.com/glossary/latest/reference/glos-chap.html#SecretAccessKey" target="_blank">secret access key</a>&nbsp;(for example,&nbsp;<code>wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY</code>). You use access keys to sign API requests that you make to AWS.</p><p> </p>